### DISCUSSION TITLE
Prevent Drop between two "TreeViewItem" in advance.

### DISCUSSION THREAD

**Discussion by a1292999652** [microsoft/microsoft-ui-xaml#7433] [Q&A]
![392d6c7aac8060c5a344e28f241744d](https://user-images.githubusercontent.com/31630317/148485783-cbaf02a4-b0a5-4159-9013-e5964b0011e9.png)
![3a3d73b8dc27ea05aa97175329dba16](https://user-images.githubusercontent.com/31630317/148485823-9f3dac7b-ed8f-4d89-9e39-cc37b1237613.png)
What event can I use to determine the target position in advance instead of processing it in "DragItemsCompleted"

**Comment by a1292999652** [comment]
"AllowDrop" and "CanDrag" of “muxc:TreeViewItem” seem to be invalid?

**Comment by a1292999652** [comment]
![f72501388b53b1334a28c08d0682c4b](https://user-images.githubusercontent.com/31630317/148508734-15697fbb-fbb7-42de-8b4f-6303466bbf81.png)
When the target node is selected, it cannot be placed

**Comment by StephenLPeters** [comment]
@ranjeshj do you know what API's are available here?
